source/admin_manual/list_of_events.rst

```diff
@@ -2726,6 +2726,91 @@ Event emitted when dict server finishes transaction. Same fields as
 :ref:`dict_transaction_finished`.
 
 
+Login proxy
+===========
+
+.. versionadded:: v2.3.18
+
+Emitted when login processes proxy a user connection to a backend.
+
+.. _pre_login_client:
+
+Pre-login Client Common fields
+------------------------------
```

Comment:
This could be in the same hierarchy level as "Login proxy". Hmm. Maybe:

```

Pre-login Client
================

.. _pre_login_client:

Client
------

... fields ...


Login proxy
===========
```